src/Extensions.cs

```diff
@@ -1,35 +1,42 @@
 using System;
-using System.IO;
+using System.Collections.Generic;
+using System.Linq;
+using Mews.Fiscalization.SignatureChecker.Model;
 
-namespace Mews.SignatureChecker
+namespace Mews.Fiscalization.SignatureChecker
 {
-    public static class Extensions
+    internal static class Extensions
     {
-        public static byte[] ReadFully(this Stream stream, bool seekToBeginning = false)
+        internal static IEnumerable<T> ToEnumerable<T>(this T value)
         {
-            if (seekToBeginning)
-            {
-                stream.Seek(0, SeekOrigin.Begin);
-            }
+            return new List<T>{value};
+        }
 
-            var buffer = new byte[32768];
-            using (var ms = new MemoryStream())
-            {
-                while (true)
-                {
-                    var read = stream.Read(buffer, 0, buffer.Length);
-                    if (read <= 0)
-                    {
-                        return ms.ToArray();
-                    }
-                    ms.Write(buffer, 0, read);
-                }
-            }
+        internal static string MkLines(this IEnumerable<string> values)
+        {
+            return $"{String.Join(Environment.NewLine, values)}";
         }
 
-        public static string ToSignatureString(this DateTime dateTime)
+        internal static string ToSignatureString(this DateTime dateTime)
         {
             return dateTime.ToString("yyyyMMddHHmmss");
         }
+
+        internal static string ToSignatureString(this Amount amount)
+        {
+            return ((int)(amount.Value * 100)).ToString();
+        }
+
+        internal static string ToSignatureString(this TaxSummary taxSummary)
+        {
+            var parts = taxSummary.Data.OrderByDescending(d => d.Key).Select(d => $"{d.Key.ToSignatureString()}:{d.Value.ToSignatureString()}");
+            return String.Join("|", parts);
+        }
```

Comment:
I would move this to the class directly. Or is there a reason to have them as extensions?